### ISSUE TITLE
E_COMPILE_ERROR was caused in line 316 

### ISSUE THREAD

**Issue by StockUnlocks** [getbowtied/shopkeeper-extender#5]
Hello,

I just updated the Shopkeeper plugin to this version:

WordPress version 5.4.2
Current theme: Shopkeeper Child (version 1.0)
Current plugin: Shopkeeper Extender (version 1.6.1)
PHP version 7.1.33

And my website fatally crashed with this error:

Error Details
=============
An error of type E_COMPILE_ERROR was caused in line 316 of the file /content/plugins/shopkeeper-extender/includes/shortcodes/wb/slider.php. Error message: Cannot declare class WPBakeryShortCode_Slider, because the name is already in use

The only way I could access my Dashboard was to use the special link that WordPress generated and emailed to me, allowing me to disable the plugin.

Thanks,

Henry

**Comment by kitserve** [comment]
This issue just bit me as well. The fix is very simple. Change line 315 of `shopkeeper-extender/includes/shortcodes/wb/slider.php` to the following:
```php
if ( class_exists( 'WPBakeryShortCodesContainer' ) and !class_exists( 'WPBakeryShortCode_Slider' ) ) {
```
Change line 319 of the same file to:
```php
if ( class_exists( 'WPBakeryShortCode' ) and !class_exists( 'WPBakeryShortCode_Image_Slide' ) ) {
```
We're just adding a check that the new class doesn't already exist. Please fix asap as this is going to break a lot of sites that have auto-updates installed.

**Comment by rthvanesa** [comment]
Hello @StockUnlocks  @kitserve ,

We are sorry for the inconvenience. We believe this error occurs only for Shopkeeper theme versions older than 2.8. However, we have applied your fix and released Shopkeeper Extender 1.6.3. 

Please let me know if you encounter any other issues. 

Have a good one!

**Comment by kitserve** [comment]
Great, thanks for the quick response! As Shopkeeper is a premium theme, many sites don't receive updates for older versions.
